packages/plugin-inline/src/Inline.ts

```diff
@@ -147,7 +148,9 @@ export class Inline<T extends JWPluginConfig = JWPluginConfig> extends JWPlugin<
 
         let inlineToCopyModifiers: VNode;
         if (range.isCollapsed()) {
-            inlineToCopyModifiers = range.start.previousSibling() || range.start.nextSibling();
+            inlineToCopyModifiers =
+                range.start.previousSibling(node => !node.test(LineBreakNode)) ||
+                range.start.nextSibling();
```

Comment:
I'm not sure I understand this fix. What it does is that it takes the first previous sibling that is *not* a `LineBreakNode`, but why does it matter ? Besides, are there no tests for this fix or are they all put together in the "tests" commit ?

Reply:
This allow the `getCurrentModifiers()` method to return the same modifiers whether `this.cache.modifiers` is empty or not.
This fix a difference of behaviour between typing a character just after a `LineBreak` or after changing the cursor position the beginning of the line and typing a character.
In the latter, the modifiers was empty.

If this is not a issue, we can remove this commit.